src/components/OperationsBar/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from 'react-bootstrap';
+
+import Operation from './Operation';
+
+export default class OperationsBar extends PureComponent {
+  static propTypes = {
+    operations: PropTypes.arrayOf(PropTypes.shape({
+      style: PropTypes.oneOf(['primary', 'success', 'info', 'warning', 'danger', 'link']),
```

Comment:
TBD should 'styles' prop be protected (= forced 'default' style) for custom/external ops (for example rewrite custom style in VIEW/container while creating operation)?
This ensures full control over 'primary' button, etc. without throwing errors in case developer does something bad.
I'm not sure.